webapp/app/js/controllers/main.js

```diff
@@ -150,6 +149,16 @@ treeherder.controller('MainCtrl', [
                 }
             });
 
+            // Shortcut: enter a custom job or platform filter
+            Mousetrap.bind('f', function(ev) {
+                // Prevent shortcut key overflow during focus
+                ev.preventDefault();
+
+                $scope.$evalAsync(
+                    $rootScope.$broadcast('click-this', "platform-job-text-search-field")
```

Comment:
Yeah, I played around with this and looks like this works:

    angular.element('#platform-job-text-search-field').trigger('click')

And then I think you don't need the directive in ``directives/main.js``